src/roswire/ros1/launch/reader.py

```diff
@@ -652,7 +656,8 @@ def locate_node_binary(self, package: str, node_type: str) -> str:
                 f"unable to locate binary for node_type [{node_type}] "
                 f"in package [{package}]"
             )
-            raise ValueError(m)
+            logger.error(m)
+            return "\\unknown"
```

Comment:
is this going to break things elsewhere?

Reply:
Honestly, I don't know. But the error will be logged if it causes problems downstream (it only happens in the case where the node can't be found, and this used to just raise an error and crash).